### PR TITLE
Romy

### PR DESCRIPTION
**Pull request by xeniter**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/91194 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4293  
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/romy.markdown [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+{% configuration %}
+host:
+  description: "The hostname or IP of your ROMY, e.g., `192.168.0.10`."
+  required: true
+  type: string
+port:
+  description: "The port where your robot interface is reachable (default: 8080)."
+  required: true
```
Comment (klaasnicolaas) [inline source/_integrations/romy.markdown]:
```suggestion
  required: true
  default: 8080
```

File: source/_integrations/romy.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+YAML configuration is around for people that prefer YAML.
+To use this integration, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+myROMY:
+  - host: IP_ADDRESS
+  - port: 8080
+  - host: Agon
+  - host: 12345678
```
Comment (klaasnicolaas) [inline source/_integrations/romy.markdown]: This seems to be incorrect YAML config

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
See also the errors that textlint returns

File: source/_integrations/romy.markdown (hunk 2 above)
Comment (xeniter) [reply source/_integrations/romy.markdown]: ups, was not aware that it didn't work at all anymore and is not wanted anymore
removed description now completly

https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision
[homeassistant.components.vacuum] The romy platform for the vacuum integration does not support platform setup. Please remove it from your config.

**Comment by frenck** [comment]
Closing PR as the parent Core PR has been closed.

../Frenck

**Pull request closed without merging**